### PR TITLE
Remove not correct annotations with node.Spec.Unschedulable

### PR DESCRIPTION
**Pull request by kerthcet**
Signed-off-by: kerthcet <kerthcet@gmail.com>

#### What type of PR is this?
/kind cleanup
/sig scheduling

#### Which issue(s) this PR fixes:

Fixes #

#### Special notes for your reviewer:
We won't remove fields from a given version, so we can't deprecate this until we have a v2 api version. 
Prefer to remove the comments directly.

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/scheduler/framework/plugins/nodeunschedulable/node_unschedulable.go [hunk 1]
```diff
@@ -68,7 +68,6 @@ func (pl *NodeUnschedulable) Filter(ctx context.Context, _ *framework.CycleState
 		Key:    v1.TaintNodeUnschedulable,
 		Effect: v1.TaintEffectNoSchedule,
 	})
-	// TODO (k82cn): deprecates `node.Spec.Unschedulable` in 1.13.
```
Comment (chendave) [inline pkg/scheduler/framework/plugins/nodeunschedulable/node_unschedulable.go:71]: iirc, this was deferred for some reasons, I am not sure remove this is the right thing to do.

Comment (kerthcet) [reply pkg/scheduler/framework/plugins/nodeunschedulable/node_unschedulable.go:71]: Some comments before https://github.com/kubernetes/kubernetes/pull/84266#issuecomment-545951881

**Pull request merged**